src/main/java/core/basesyntax/model/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.controller.ConsoleHandler;
+import core.basesyntax.model.dao.BetDao;
+import core.basesyntax.model.dao.UserDao;
+import core.basesyntax.model.factory.Factory;
+import core.basesyntax.model.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
+
+public class Main {
+    public static void main(String[] args) throws InvocationTargetException,
+            NoSuchMethodException, InstantiationException,
+            IllegalAccessException, RuntimeException {
```

Comment:
```suggestion
    public static void main(String[] args) throws InvocationTargetException,
            NoSuchMethodException, InstantiationException,
            IllegalAccessException {
```
RuntimeExceptions should not be indicated in method signature

Reply:
done